test/clj/rems/test_performance.clj

```diff
@@ -0,0 +1,94 @@
+(ns ^:integration rems.test-performance
+  (:require [clj-memory-meter.core :as mm]
+            [medley.core :refer [map-vals]]
+            [mount.core :as mount]
+            [rems.api.applications-v2 :as applications-v2]
+            [rems.api.reviews :as reviews]
+            [rems.db.dynamic-roles :as dynamic-roles]))
+
+(defn duration-millis [f]
+  (let [start (System/nanoTime)
+        _ (f)
+        end (System/nanoTime)]
+    (/ (- end start) 1000000.0)))
+
+(defn run-benchmark [benchmark iterations]
+  ((:setup benchmark))
+  (doall (for [_ (range iterations)]
+           (duration-millis (:benchmark benchmark)))))
+
+(defn statistics [measurements]
+  (let [measurements (sort measurements)]
+    {:min (first measurements)
+     :median (nth measurements (quot (count measurements) 2))
+     :max (last measurements)}))
+
+(defn run-benchmarks [iterations benchmarks]
+  ;; warmup
+  (doseq [benchmark benchmarks]
+    (run-benchmark benchmark iterations))
+  ;; actual measurements
+  (doseq [benchmark benchmarks]
+    (let [measurements (run-benchmark benchmark iterations)]
+      (println (:name benchmark)
+               (->> (statistics measurements)
+                    (map-vals #(format "%.3fms" %)))))))
+
+(comment
```

Comment:
Instead of a huge comment, how about having the helpers and benchmarks defined in the ns, and only commenting the run-benchmarks invocations?